fission-cli/library/Fission/CLI/Handler/User/Register.hs

```diff
@@ -85,10 +83,11 @@ createAccount ::
   , m `Raises` AlreadyExists Ed25519.SecretKey
   , m `Raises` Username.Invalid
   , Show (OpenUnion (Errors m))
+  , Contains (Errors m) (Errors m)
```

Comment:
This line looks weird, right? Of course `Contains a a`!

Reply:
Well, it's not an assertion that `Errors m` contains itself. It loops through the elements at compile time, and makes sure that they match. We don't depend on ordering, so the `Contains` constraint happens to be the thing that does that. I've tried to get rid of it, because it reads weird, but alas...